diff --git a/node_modules/@pancakeswap-libs/uikit/src/theme/colors.ts b/node_modules/@pancakeswap-libs/uikit/src/theme/colors.ts
index 4601410..1f78e6d 100644
--- a/node_modules/@pancakeswap-libs/uikit/src/theme/colors.ts
+++ b/node_modules/@pancakeswap-libs/uikit/src/theme/colors.ts
@@ -1,13 +1,13 @@
 import { Colors } from "./types";
 
 export const baseColors = {
-  failure: "#ED4B9E",
-  primary: "#aa8929",
-  primaryBright: "#aa8929",
-  primaryDark: "#aa8929",
-  secondary: "#7645D9",
-  success: "#31D0AA",
-  warning: "#FFB237",
+  failure: "#ff0000",
+  primary: "#ff0000",
+  primaryBright: "#ff0000",
+  primaryDark: "#ff0000",
+  secondary: "#ff0000",
+  success: "#ff0000",
+  warning: "#ff0000",
 };
 
 export const brandColors = {
@@ -17,17 +17,17 @@ export const brandColors = {
 export const lightColors: Colors = {
   ...baseColors,
   ...brandColors,
-  background: "#FAF9FA",
-  backgroundDisabled: "#E9EAEB",
-  contrast: "#191326",
-  invertedContrast: "#FFFFFF",
-  input: "#eeeaf4",
-  tertiary: "#EFF4F5",
-  text: "#452A7A",
-  textDisabled: "#BDC2C4",
-  textSubtle: "#8f80ba",
-  borderColor: "#E9EAEB",
-  card: "#FFFFFF",
+  background: "#ff0000",
+  backgroundDisabled: "#ff0000",
+  contrast: "#ff0000",
+  invertedContrast: "#ff0000",
+  input: "#ff0000",
+  tertiary: "#ff0000",
+  text: "#ff0000",
+  textDisabled: "#ff0000",
+  textSubtle: "#ff0000",
+  borderColor: "#ff0000",
+  card: "#ff0000",
   gradients: {
     bubblegum: "linear-gradient(139.73deg, #E6FDFF 0%, #F3EFFF 100%)",
   },
